src/main/kotlin/org/rust/lang/core/resolve/ResolveEngine.kt

```diff
@@ -212,6 +209,34 @@ object ResolveEngine {
         return module.project.getPsiFor(module.cargoWorkspace?.findCrateByName(name)?.crateRoot)?.rustMod
     }
 
+    fun resolveLabel(label: RsLabel): RsLabelDecl? =
+        label.branch
+            .mapNotNull {
+                when (it) {
+                    is RsLoopExpr -> it.labelDecl
+                    is RsForExpr -> it.labelDecl
+                    is RsWhileExpr -> it.labelDecl
```

Comment:
An interface. If the name of interface methods match accessors generated by grammar kit, everything will work automatically.